Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestRunIndexViewController.swift

```diff
@@ -9,23 +9,24 @@
 import UIKit
 
 class TestRunIndexViewController: UIViewController {
+   
     @IBOutlet weak var cancelRun: UIBarButtonItem!
+    @IBOutlet weak var testRunNameLabel: UILabel!
+    @IBOutlet weak var testStepTable: UITableView!
     
     var instance = ""
     var username = ""
     var password = ""
     var runId = -1
     var runName = ""
     
-    @IBOutlet weak var runNameLabel: UILabel!
-    @IBOutlet weak var runIdLabel: UILabel!
-    
     override func viewDidLoad() {
         super.viewDidLoad()
         //hide the default back button and instead show cancel run
         self.navigationItem.hidesBackButton = true
-        runNameLabel.text = "Name: " + runName
-        runIdLabel.text = "Run ID: " + String(runId)
+        testRunNameLabel.text = runName
```

Comment:
This is not hardcoded, the previous screen loads the run name and this displays that name at the top of the screen.